docs/validators/state-sync.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 3
+sidebar_label: "Option 1: State-Sync"
+---
+
+# Syncing a Node Using State-Sync
+
+:::note
+
+This guide assumes you've completed the
+"[Setting Up Your Node](/validators/setup)" step.
+
+:::
+
+Follow this guide to setup a RPC node using state-sync. This guide was created
+and tested in MacOS and may need to be modified slightly for your local system.
+
+## State-sync nodes
+
+| Network      | State Sync Server |
+| :----------- | :---------------- |
+| Mainnet Beta | 34.28.196.79      |
+| Testnet      | 34.30.34.119      |
```

Comment:
Should these instead be FQDNs? @gzukel @julianrubino

Reply:
This is probably fine for now but we can do another PR with a list of alternatives before we kill off our state sync nodes